README.md

```diff
@@ -1,8 +1,362 @@
 # CUDA C++ Core Libraries (CCCL)
 
-The future home of the CUDA C++ Core Libraries (Thrust, CUB, libcu++).
+Welcome to the CUDA C++ Core Libraries (CCCL) where our mission is to make CUDA C++ more delightful.
+
+This repository unifies three essential CUDA C++ libraries into a single, convenient repository: 
+
+1. [Thrust](https://github.com/nvidia/thrust)
+2. [libcudacxx](https://github.com/nvidia/libcudacxx)
+3. [CUB](https://github.com/nvidia/cub)
+
+Our goal is to provide CUDA C++ developers with the building blocks they need to make it easier to write safe and efficient code. By bringing these libraries together, we hope to streamline your development process and broaden your ability to leverage the power of CUDA C++.
+For more information about our decision to unify these projects, see our announcement here: (TODO)
```

Comment:
I'm assuming the TODO will be populated later

Reply:
Yeah, this is a bit of a circular dependency where the announcement will link to the README and the README will link to the announcement...